tests/multipleBackend/routes/routeBackbeat.js

```diff
@@ -37,12 +37,12 @@ const backbeatAuthCredentials = {
     accessKey: accessKeyId,
     secretKey: secretAccessKey,
 };
-const TEST_BUCKET = 'backbeatbucket';
-const TEST_ENCRYPTED_BUCKET = 'backbeatbucket-encrypted';
+const TEST_BUCKET_PREFIX = 'backbeatbucket';
+const TEST_ENCRYPTED_BUCKET_PREFIX = 'backbeatbucket-encrypted';
 const TEST_KEY = 'fookey';
-const NONVERSIONED_BUCKET = 'backbeatbucket-non-versioned';
-const VERSION_SUSPENDED_BUCKET = 'backbeatbucket-version-suspended';
-const BUCKET_FOR_NULL_VERSION = 'backbeatbucket-null-version';
+const NONVERSIONED_BUCKET_PREFIX = 'backbeatbucket-non-versioned';
+const VERSION_SUSPENDED_BUCKET_PREFIX = 'backbeatbucket-version-suspended';
```

Comment:
With uuid v4 this is already 70 bytes long, e.g. `backbeatbucket-version-suspended-ae718b1c-a677-48e5-be1b-065706c06e8f`, i think the bucket name limit is 63 characters, so I'm not sure to understand how it can work? AT lkeast we should make is smaller